testing/benchmark/Makefile

```diff
@@ -2,23 +2,25 @@ APMBENCH_PATH ?= ../../systemtest/cmd/apmbench
 APMBENCH_GOOS ?= linux
 APMBENCH_GOARCH ?= amd64
 
+TFVARS_SOURCE ?= terraform.tfvars.example
+
 BENCHMARK_WARMUP ?= 10000
 BENCHMARK_AGENTS ?= 64
 BENCHMARK_COUNT ?= 3
 BENCHMARK_TIME ?= 2m
 BENCHMARK_RUN ?= Benchmark
 BENCHMARK_RESULT ?= benchmark-result.txt
-BENCHMARK_DETAILED ?= false
+BENCHMARK_DETAILED ?= true
 
-GOBENCH_INDEX ?= gobench
+GOBENCH_INDEX ?= apmbench-v2
 GOBENCH_USERNAME ?= admin
 GOBENCH_PASSWORD ?= changeme
 GOBENCH_HOST ?= http://localhost:9200
 GOBENCH_DEFAULT_TAGS = apm_server_version=$(APM_SERVER_VERSION)
 
 SSH_USER ?= ec2-user
 SSH_OPTS ?= -o LogLevel=ERROR -o StrictHostKeyChecking=no
-SSH_KEY ?= ~/.ssh/id_rsa_terraform
+SSH_KEY ?= ./id_rsa_terraform
```

Comment:
Would you mind reverting this? We're already using this environment ad-hoc and changing the defaults may cause some problems.